python/ingestion/standardized_columns.py

```diff
@@ -32,7 +32,6 @@
 INCOME_COL = "income"
 POPULATION_PCT_COL = "population_pct"
 
-TOTAL_VALUE = "Total"
```

Comment:
I'm not super clear how we will now access rows in our data sources that use the string `TOTAL` or `Total`. Doesn't the ACS data use total and not all? I know with the 2010 data we aren't pinging the API directly and just are getting it from `/data` but for regular ACS won't it need "Total"? Or is it always summing the groups to create an all/total row?

Reply:
Right, if you notice [here](https://github.com/SatcherInstitute/health-equity-tracker/pull/1558/files?diff=unified&w=0#diff-47d9e5580ee35db4b25d4a9f0b6a25a9334ef7c31e9257062178a013418352bfL39-R44) in `python/datasources/acs_population.py`, we map the value of whatever represents the "Total" value from the ACS API to whatever our `Total/All` value is. So we are simply changing that map from `Total` to `All`